src/styles/blocks/container.scss

```diff
@@ -0,0 +1,12 @@
+.container {
+  margin: 0 auto;
+  padding: 0 20px;
+
+  @include onTablet {
+    padding: 0 72px;
+  }
+
+  @include onDesktop {
```

Comment:
It is recommended to have a single space before the opening curly brace of a nested block for better readability 📖.